src/css/application.css

```diff
@@ -420,13 +421,13 @@ body {
 }
 
 .editors__column-divider {
-  background-color: var(--color-dark-gray);
+  background-color: #aaa;
```

Comment:
I actually considered this but then wasn't sure what to name it..? like `--color-dark-gray-lighter`? heh. Any thoughts on that one?